internal/runners/projects/projects.go

```diff
@@ -18,15 +21,32 @@ type projectWithOrg struct {
 	Name           string   `json:"name"`
 	Organization   string   `json:"organization"`
 	LocalCheckouts []string `json:"local_checkouts,omitempty"`
+	Executables    []string `json:"executables,omitempty"`
 }
 
 type projectWithOrgs []projectWithOrg
 
+func newProjectWithOrg(name, org string, checkouts []string) *projectWithOrg {
```

Comment:
I don't think you'll need the pointer return type here as the slice value is a pointer to the backing array.

Reply:
You have eagle eyes. Thanks for noticing that.